schema/artwork/index.js

```diff
@@ -39,6 +39,10 @@ import {
   GraphQLInt,
 } from 'graphql';
 
+const is_inquireable = ({ inquireable, acquireable }) => {
+  return (inquireable && !acquireable);
+};
```

Comment:
I think at some point the site would have both 'inquire' and 'purchase' buttons for e-commerce works. So they were definitely not mutually exclusive.